Gemfile

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
+
+# gem "rails"
+
+gem "jekyll", "~> 4.2"
```

Comment:
GitHub Pages currently supports Jekyll 3.9.2: https://pages.github.com/versions/ I don't think it will make much difference in this case because the remaining dependencies are supported and it isn't a particularly complicated site. However, I'm a little uneasy using a major version different from GitHub Pages, although I don't think it will break during the build on GitHub pages. 

I think there are three options to resolve it:

1. Ignore the dependency difference, merge and see if it builds correctly, we can always rollback if it doesn't
2. Use the GitHub pages gem which uses a compatible version of Jekyll (added as a suggestion, but ignore if we aren't going for this option). The downside is that it may have out of date dependencies, nullifying the point of the updates. 
3. Configure a dedicated GitHub Action workflow to build outside the default GitHub pages workflow and use the specified version of Jekyll

Suggestions/comments welcome on the preferable option. I'm tempted to try 1 with the fallback of if the build fails rolling back and then choosing between option 2 or 3. 

@pospeselr, @opsyne any thoughts/preferences?

```suggestion
gem "github-pages", "~> 227", group: :jekyll_plugins
```

Reply:
i think using the "github-pages" gem is best. it seems like the easiest way to get dev builds and github-pages builds playing nicely